examples/dataview/category-data-view-all-modes.html

```diff
@@ -94,9 +94,9 @@
       );
     const errorContainer = document.getElementById('errorContainer');
 
-    let categoryRemote;
-    let categoryRemoteFiltered;
-    let categoryLocal;
+    let categoryGlobal;
+    let categoryViewportPrecise;
```

Comment:
I think we should just keep global & viewport for public examples (not entering into the 'fast' vs 'precise' modes at all. I would like to ask you if you please could remove that duality. 

And if you see something relevant for us in this example, move it to a debug example. Could you please?